corehq/apps/translations/integrations/transifex/views.py

```diff
@@ -1,28 +1,19 @@
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
 
-from io import open
+from io import BytesIO, open
 from zipfile import ZipFile
 
-import openpyxl
-import polib
-from io import BytesIO
-from corehq.apps.translations.integrations.transifex.transifex import Transifex
-from corehq.apps.translations.integrations.transifex.utils import transifex_details_available_for_domain
-from corehq.apps.translations.utils import get_file_content_from_workbook
-
 from django.contrib import messages
 from django.http import HttpResponse
 from django.shortcuts import redirect
 from django.urls import reverse
 from django.utils.decorators import method_decorator
-from django.utils.translation import (
-    ugettext as _,
-    ugettext_noop,
-    ugettext_lazy,
-)
+from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy, ugettext_noop
+
+import openpyxl
+import polib
```

Comment:
should this be on the top?

Reply:
(ditto [isort](https://isort.readthedocs.io/en/stable/))